LibreNMS/OS/Fortios.php

```diff
@@ -38,7 +38,6 @@ public function discoverOS(Device $device): void
     {
         parent::discoverOS($device); // yaml
 
-        $device->hardware = $device->hardware ?: $this->getHardwareName();
```

Comment:
You've only removed some of the values form the `getHardwareName()` function so this is probably still needed isn't it?

Reply:
It is used by some other device

My idea is to remove this file